tests/functionals/TestPageBase.py

```diff
@@ -44,5 +45,27 @@ def test_003_page_base_method_go_page_url(self):
         finally:
             self.bot.close()
 
+    def test_004_page_base_method_get_elements_one_element(self):
+        try:
+            selectors = [".logo-dark[alt='Netzulo Testing Lab']"]
+            self.bot = BotBase(BotConfig(nose_config=cfg))
+            page = PageBase(self.bot,cfg['TEST_UNITARIES']['url'],selectors=selectors)
+            assert isinstance(page.elements[0], WebElement)
+        except Exception as err:
```

Comment:
Why the `except`? After all you're rasing a new exception with the same error but overwriting the traceback, which is the most useful part of an exception.

If you did it to use `finally`, it can be used in a `try` even when it have no `except` clause.

Same in line 65.

OTOH maybe you can replace entirely the `try/finally` clause with something like this in the teardown of the tests:
```
if self.bot:
    self.bot.close()
```

Reply:
ahahahhahaahahaha true, but need to improve TestInfoBase class with inherit class *TestInfoBot* what can use bot instance to open and close at *setUp* , *tearDown* test methods... but....

when execute tests,

1) need to use 1 bot for all tests in this testsuite
2) need to use multiple bots for each tests
3) last combination with parrallel selenium servers

im on it, you have reason, must implement on *tearDown* method 🥇

Reply:
related to #48